core/arch/arm/plat-rzn1/main.c

```diff
@@ -0,0 +1,72 @@
+// SPDX-License-Identifier: BSD-2-Clause
+/*
+ * Copyright (c) 2017, Schneider Electric
+ * Copyright (c) 2020, Linaro Limited
+ */
+
+#include <arm.h>
+#include <console.h>
+#include <drivers/gic.h>
+#include <drivers/ns16550.h>
+#include <kernel/boot.h>
+#include <kernel/interrupt.h>
+#include <kernel/panic.h>
+#include <mm/core_memprot.h>
+#include <mm/core_mmu.h>
+#include <platform_config.h>
+#include <rzn1_tz.h>
+
+static struct gic_data gic_data;
+static struct ns16550_data console_data;
+
+register_phys_mem(MEM_AREA_IO_SEC, GIC_BASE, CORE_MMU_PGDIR_SIZE);
+register_phys_mem(MEM_AREA_IO_SEC, PERIPH_REG_BASE, CORE_MMU_PGDIR_SIZE);
+register_ddr(DRAM_BASE, DRAM_SIZE);
+
+void console_init(void)
+{
+	ns16550_init(&console_data, CONSOLE_UART_BASE, IO_WIDTH_U32, 2);
+	register_serial_console(&console_data.chip);
+}
+
+void main_init_gic(void)
+{
+	vaddr_t gicc_base;
+	vaddr_t gicd_base;
```

Comment:
minor: prefer with an initialization value, for consistency and [coding standards](https://optee.readthedocs.io/en/latest/general/coding_standards.html#coding-standards).
Ditto in `rzn1_tz_init()`.

Reply:
Okay.